### PR TITLE
Move crispy style to hqwebapp

### PR DESCRIPTION
**Pull request by orangejenny**

### CODE REVIEW

**Comment by orangejenny** [comment]
code buddies @gcapalbo / @calellowitz

File: corehq/apps/hqwebapp/crispy.py [hunk 1]
```diff
@@ -96,5 +101,257 @@ def render(self, form, form_style, context, template_pack=None):
...
+    actually be responsive (col-lg-offset-*, col-md-offset-*, etc). Also includes
+    support for static controls.
+    todo since we forked crispy forms, this class is no longer necessary. http://manage.dimagi.com/default.asp?186372
+    """
+    template = 'hqwebapp/crispy/field.html'
+
+    def __init__(self, *args, **kwargs):
+        self.is_static = False
+        if 'is_static' in kwargs:
+            self.is_static = kwargs.pop('is_static')
```
Comment (gcapalbo) [inline corehq/apps/hqwebapp/crispy.py]: (could have also done `self.is_static = kwargs.pop('is_static', False)`)

Comment (gcapalbo) [reply corehq/apps/hqwebapp/crispy.py]: oh, i see this was probably just moved from the original file

**Review by gcapalbo — APPROVED** [review]

**Comment by orangejenny** [comment]
Updated to fix two fields that were defined in both the `hqwebapp` and `style` versions of crispy.

**Review by gcapalbo — APPROVED** [review]

**Pull request merged**